shared/logger.go

```diff
@@ -54,37 +54,26 @@ func NewNilLogger() Logger {
 	return nl
 }
 
-type gaeLogger struct {
-	ctx context.Context
-}
-
-func (l gaeLogger) Debugf(format string, args ...interface{}) {
-	gaelog.Debugf(l.ctx, format, args...)
-}
-
-func (l gaeLogger) Errorf(format string, args ...interface{}) {
-	gaelog.Errorf(l.ctx, format, args...)
-}
-
-func (l gaeLogger) Infof(format string, args ...interface{}) {
-	gaelog.Infof(l.ctx, format, args...)
-}
-
-func (l gaeLogger) Warningf(format string, args ...interface{}) {
-	gaelog.Warningf(l.ctx, format, args...)
-}
-
-// newGAELogger returns a Google App Engine Standard Environment logger bound to
-// the given context.
-func newGAELogger(ctx context.Context) Logger {
-	return gaeLogger{ctx}
-}
-
 // NewAppEngineContext creates a new Google App Engine Standard-based
 // context bound to an http.Request.
 func NewAppEngineContext(r *http.Request) context.Context {
```

Comment:
Have to [close the client](https://pkg.go.dev/cloud.google.com/go/logging#hdr-Closing_your_Client)